src/map/script.c

```diff
@@ -19115,6 +19161,68 @@ BUILDIN_FUNC(instance_check_party)
 }
 
 /*==========================================
+ * instance_check_guild
+ * Values:
+ * guild_id : Guild ID of the invoking character. [Required Parameter]
```

Comment:
If it's the guild id of the invoking character, why do we have it marked as required?
We could just read it from the attached player and have it optional.